packages/react-tinacms-editor/src/plugins/Image/Menu/ProsemirrorMenu.tsx

```diff
@@ -65,8 +65,7 @@ export const ProsemirrorMenu = ({ uploadImages, mediaDir }: MenuProps) => {
 
   async function onMediaSelect(media?: Media) {
     if (media) {
-      const previewSrc = await cms.media.previewSrc(media.id)
-      insertImageInEditor(previewSrc)
+      insertImageInEditor(media.id)
```

Comment:
this should call the parse method instead of passing the id thru directly

Reply:
I think that mayyybe the parse is handled in that `useImageProps` hook on `upload`?